samples/nrf9160/modem_shell/overlay-rtt.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_USE_SEGGER_RTT=y
+
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+
+CONFIG_SHELL_BACKEND_RTT=y
+CONFIG_SHELL_PROMPT_RTT="mosh:~$ "
+CONFIG_SHELL_BACKEND_SERIAL=n
+CONFIG_SHELL_PROMPT_UART=""
```

Comment:
`CONFIG_SHELL_PROMPT_UART` is cleared to remove a Kconfig warning. There's still one warning caused by `CONFIG_SHELL_BACKEND_SERIAL_RX_RING_BUFFER_SIZE` with the RTT overlay. The problem is that this option is set in prj.conf and at least I don't know how I could unset an integer option when RTT is used instead of UART.

If we want to get rid of the warning, we could move it to Kconfig instead, I guess.